hist/hist/test/test_TH1_SaveAs.cxx

```diff
@@ -0,0 +1,220 @@
+#include "gtest/gtest.h"
+
+#include "TString.h"
+#include "TH1.h"
+#include "TSystem.h"
+#include <fstream>
+#include <iostream>
+#include <sstream>
+#include <string>
+
+struct TestSaveAs {
+   const Int_t Nbins = 5;
+   const Int_t N = Nbins+2;  // 0 and N are the under/overflow bins, resp.
+   TString fnam = "H1dump.";
+
+   void SaveHist(const TString &myext, const TString &myoption="") {
+      // Bin contents and bin errors
+      Double_t binc[N] = {5.2, 0, 10.8, 12.3, 9.5, 7.3, 15.2};
+      Double_t bine[N] = {2.1, 0,  2.5,  2.1, 3.5, 2.7,  4.7};
```

Comment:
I saw in the CI a few errors such as 
```
  Error: /Users/sftnight/ROOT-CI/src/hist/hist/test/test_TH1_SaveAs.cxx:19:21: error: variable-sized object may not be initialized
        Double_t binc[N] = {5.2, 0, 10.8, 12.3, 9.5, 7.3, 15.2};
```

On our Mac nodes. This is because clang doesn't allow initializing an array with dimension given by a variable which is not usable in a constexpr context.

This can be fixed by converting `Nbins`, `N` and any other variable used to initialize arrays to `constexpr` instead of `const`.

Reply:
Done. Had to make them ```static constexpr``` or would not compile.